### PR TITLE
Revise README

### PR DESCRIPTION
**Pull request by hodog**
Revised some of the writing in README to make it more understandable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 
-## 
+##
 <p align="center">
   <img alt="Reactide Screenshot" src="http://reactide.io.s3-website-us-west-1.amazonaws.com/images/reactide-screenshot2.png">
 </p>
 
 ## Get right to coding
-Reactide runs an integrated Node server and custom browser simulator, which eliminates the need to configure servers, build-tools, and even offers hot module reloading right out of the box. Projects developed in Reactide are build-tool agnostic. As projects evolve, the developer only needs to add necessary dependencies without having to make decisions before coding has even started.
+Reactide runs an integrated Node server and custom browser simulator. Configuring servers and build-tools are eliminated, and hot module reloading is offered right out of the box. Projects developed in Reactide are build-tool agnostic. As projects evolve, the developer only needs to add necessary dependencies without having to make decisions before coding has started.
```
Comment (sergiusignacius) [inline README.md]: Should be _is_ eliminated?

Comment (hodog) [inline README.md]: I think it depends on how you want to frame the sentence. But for that specific sentence, no, it should not be eliminated.

Comment (sergiusignacius) [inline README.md]: Just to be clear, I was suggesting:

> Configuring servers and build-tools is eliminated

instead of are eliminated. Just feels weird to me read that. But hey, I may be nitpicking. Good job nonetheless. 👍

Comment (hodog) [inline README.md]: Wait, sorry about that I didn't know which "is" you were referring to. In that case, I think you might be right about that one. Good catch.

Comment (hodog) [inline README.md]: By the way, do you want me to make the change and commit again? I've only started doing this and I'm not sure.

Comment (sergiusignacius) [inline README.md]: If you do the change (on your fork) and commit it, it'll show up here.

Comment (hodog) [inline README.md]: I don't have to make another pull request right?

Comment (sergiusignacius) [inline README.md]: Right.
